src/dextrader/nomics/utils.py

```diff
@@ -0,0 +1,33 @@
+import urllib.request
+import json
+import os
+import pandas as pd
+
+
```

Comment:
More a note for me when I get working on this.. but maybe adding either docstring and/or [typing hints](https://docs.python.org/3.7/library/typing.html#module-typing) to the function parameters for the hints to work in IDE/jupyter.

Reply:
added typehints

Reply:
Awesome!